trace/backend_collector.go

```diff
@@ -272,6 +276,64 @@ func (c *BackendCollector) processTLSHandshake(tls akinet.TLSHandshakeMetadata)
 	return nil
 }
 
+func (c *BackendCollector) processClientTimeout(packet akinet.ParsedNetworkTraffic, content akinet.ClientTimeoutMetadata) error {
+	witnessID := learn.ToWitnessID(content.StreamID, content.Seq)
+	val, ok := c.pairCache.LoadAndDelete(witnessID)
+
+	if !ok {
+		// this should not have happened,a partial witness should have existed for the timeout data passed to us
+		return errors.Errorf("no partial witness found for client timeout %v", witnessID)
+	}
+
+	pair := val.(*witnessWithInfo)
+	if pair.witness.Method.GetMeta() != nil {
+		meta := pair.witness.Method.GetMeta()
+		if meta != nil {
```

Comment:
You can do this all on one line:

```suggestion
	if meta := pair.witness.Method.GetMeta(); meta != nil {
```